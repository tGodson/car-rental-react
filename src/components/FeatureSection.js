import React from 'react';
import { Row, Col } from 'react-bootstrap';
import car from '../imgs/feature-ride.png';

export default function FeatureSection() {
  return (
    <section className="about-section feature-section">
      <Row className="mt-2">
        <Col sm={6}>
          <img src={car} className="img-fluid" alt="ride" />
        </Col>
        <Col sm={6} className="pt-5">
          <h4 className="about-section__left-title pt-5">
            Best rides for your trips
          </h4>
          <p className="about-section__desc">
            More than hundreds of rides at your disposal. Only the best and checked with owners.
            We will find best place for you and conditions for a safe parking.
          </p>
          <p>
            <a href="#register" className="btn btn-oval">
              Learn more
            </a>
          </p>
        </Col>
      </Row>
    </section>
  );
}
