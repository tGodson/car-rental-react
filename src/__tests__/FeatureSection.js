import React from 'react';
import renderer from 'react-test-renderer';
import FeatureSection from '../components/FeatureSection';

it('renders FeatureSection correctly', () => {
  const tree = renderer.create(<FeatureSection />);
  expect(tree).toMatchSnapshot();
});
