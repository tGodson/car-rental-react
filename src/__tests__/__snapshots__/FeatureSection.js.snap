// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders FeatureSection correctly 1`] = `
<section
  className="about-section feature-section"
>
  <div
    className="mt-2 row"
  >
    <div
      className="col-sm-6"
    >
      <img
        alt="ride"
        className="img-fluid"
        src="feature-ride.png"
      />
    </div>
    <div
      className="pt-5 col-sm-6"
    >
      <h4
        className="about-section__left-title pt-5"
      >
        Best rides for your trips
      </h4>
      <p
        className="about-section__desc"
      >
        More than hundreds of rides at your disposal. Only the best and checked with owners. We will find best place for you and conditions for a safe parking.
      </p>
      <p>
        <a
          className="btn btn-oval"
          href="#register"
        >
          Learn more
        </a>
      </p>
    </div>
  </div>
</section>
`;
